lifecycleScripts/preinstall.js

```diff
@@ -9,12 +9,26 @@ module.exports = function prepareForBuild() {
 
   return exec("npm -v")
     .then(function(npmVersion) {
-      if (npmVersion.split(".")[0] < 3) {
-        console.log("[nodegit] npm@2 installed, pre-loading required packages");
-        return exec("npm install --ignore-scripts");
-      }
+    if (npmVersion.split(".")[0] < 3) {
+      console.log("[nodegit] npm@2 installed, " + 
+                  "pre-loading required packages");
+      return exec("npm install --ignore-scripts");
+    }
+
+    return Promise.resolve();
```

Comment:
This return lines up with the indentation of its closing brace.

Would you kindly update the indentation from lines 12 to 18 to be at least 6?

Reply:
Updated!